ts/features/zendesk/screens/ZendeskAskPermissions.tsx

```diff
@@ -210,7 +209,6 @@ const ZendeskAskPermissions = (props: Props) => {
     deviceDescription: `${getModel()} · ${
       isIos ? "iOS" : "Android"
     } · ${getSystemVersion()}`,
-    currentVersion: getAppVersion(),
```

Comment:
Is there a reason why this has been moved out? Just curious.

Reply:
As far as I can see, if I leave that there I have to add an item inside `itemProps` with a `value` and a `zendeskId` to match the field in the Zendesk dashboard, and doing that would basically show the old `Version of the app in use` field to the user. What I did is: replace that old field with a new one related to the history of versions, sending anyway the current version "outside" of the `itemProps` object for backward compatibility!

If I'm missing something or you see a better way to do that (maybe with less changes too) please let me know 🙂

Reply:
Nope, I guess it's ok.